client/src/routes/Session/hooks/useUpdateSession.spec.ts

```diff
@@ -38,23 +38,4 @@ describe('useUpdateSession', () => {
       expect(fetchMock).toHaveBeenCalledTimes(1);
     });
   });
-
-  describe('setEnded', () => {
```

Comment:
hmm weird, don't remember removing this or having any reason for it, will add it back

Reply:
[9bde71c](https://github.com/29ki/29k/pull/927/commits/9bde71cce63f37e7f8a033d4bc2893eb62dfeaa1)